### ISSUE TITLE
Can not upload image inside code block

### ISSUE THREAD

**Issue by thaind97git** [netlify/netlify-cms#3038]
**Describe the bug**
When I try to upload an image inside Code Block. I received an error is: 
```
Uncaught (in promise) TypeError: Cannot read property 'find' of undefined
    at mediaLibrary.js:299
    at index.js:8
    at redux.js:475
    at MediaLibrary.js:151
    at Object.Gg (react-dom.production.min.js:155)
    at Fg (react-dom.production.min.js:13)
    at Ig (react-dom.production.min.js:13)
    at be (react-dom.production.min.js:15)
    at Jg (react-dom.production.min.js:156)
    at Ic (react-dom.production.min.js:15)
````

**To Reproduce**
1. Go to localhost:8000/admin
2. Click on New Blog
3. Click on new Image inside Code Block
3. Try to upload an image
4. See error at console

**Expected behavior**
I want to can upload an image without any error

**Screenshots**

![image](https://user-images.githubusercontent.com/42630357/71863040-3410e300-312e-11ea-84e6-9c23689e81aa.png)

**Applicable Versions:**
 - Netlify CMS app version: [2.10.1]
 - Git provider: [GitHub]
 - OS: [Windows 10]
 - Browser version: [79.0.3945.88]
 - Node.JS version: [v10.16.0]

**CMS configuration**
```
backend:
  name: git-gateway
  branch: master

media_folder: static/img
public_folder: /img
publish_mode: editorial_workflow

collections:
  - name: "blog"
    label: "Blog"
    folder: "src/blog-posts"
    create: true
    slug: "{{year}}-{{month}}-{{day}}-{{slug}}"
    editor:
     preview: true
    fields:
      - { label: "Title", name: "title", widget: "string" }
      - {
          label: 'Publish Date',
          name: 'date',
          widget: 'datetime',
          dateFormat: 'YYYY-MM-DD',
          timeFormat: 'HH:mm',
          format: 'YYYY-MM-DD HH:mm',
        }
      - { label: "Description", required: false, name: "description", widget: "string" }
      - { label: 'Body', name: 'body', widget: 'markdown', hint: 'Main content goes here.' }
      - { 
          label: "Tags", 
          name: "tags", 
          widget: "select", 
          options: ['ReactJs', 'JavaScript', 'Redux', 'NodeJs'],
          multiple: true 
        }
```
My repo: https://github.com/thaind97git/tech-blog

**Comment by erquhart** [comment]
There isn’t a way to attempt uploading an image to a code block - can you describe the specific steps that you’re taking to produce this error?

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by thaind97git** [comment]
Hi @erquhart sorry about that. I forget that step and now I updated it. Can you help me check again?

**Comment by erquhart** [comment]
It isn’t possible to upload an image to a code block, so I really don’t know how to go about reproducing. A loose description of what you remember, maybe?

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by thaind97git** [comment]
Hi @erquhart , When I try to login with this site: https://deploy-preview-1433--cms-demo.netlify.com (get from [#1433](https://github.com/netlify/netlify-cms/pull/1433)), I can upload an image inside Code Block.
So, with my case, I don't know exactly why error appears.
You can clone my repository and follow with specific steps:

1. Run `yarn dev`
2. open `localhost:8000/admin`
3. click on `New Blog` button
4. inside Body ( widget: 'markdown' ) click on `+ Image`
5. Try to upload an image and you will see an error inside the console.

All the code, I just follow with [this Turorial](https://www.netlifycms.org/docs/start-with-a-template/).

**Comment by erquhart** [comment]
Just released 2.10.3, can you update and see if it still happens?

**Comment by thaind97git** [comment]
@erquhart , I resolved this issue by an update to 2.10.3. Thank for your support.
